libnmstate/nm/wired.py

```diff
@@ -39,6 +41,9 @@ def create_setting(iface_state, base_con_profile):
     if not wired_setting:
         wired_setting = nmclient.NM.SettingWired.new()
 
+    if mac:
+        wired_setting.props.cloned_mac_address = mac
```

Comment:
should be check that it does not contains special NM values like random or stable?

Reply:
The schema accepts only a mac format.